pyravendb/raven_operations/timeseries_operations.py

```diff
@@ -0,0 +1,161 @@
+from typing import Optional, Iterable, List, Generator
+from pyravendb.commands.raven_commands import *
+from pyravendb.tools.utils import Utils
+from .operations import Operation
+from datetime import datetime
+
+import sys
+
+
+class TimeSeriesRange:
+    def __init__(self, name, from_date: Optional[datetime] = None, to_date: Optional[datetime] = None):
+        if not name:
+            raise ValueError("TimeSeriesRange must include name")
+        self.name = name
+        self.from_date = from_date
```

Comment:
This should default to `0001-01-01`

Reply:
not needed if you don't send any will be automatically like this 
{'From': '0001-01-01T00:00:00.0000000Z', 'To': '9999-12-31T23:59:59.9999999Z'} example from the code

Reply:
How does this work? Both of them are: `Optional[datetime] = None` and are used with:

`from={Utils.datetime_to_string(range_.from_date)}`

Reply:
Oh, wait, this happens on the server side, no?